app/Http/Controllers/Admin/UserController.php

```diff
@@ -5,17 +5,29 @@
 use App\Http\Controllers\Controller;
 
 use App\Http\Requests\Admin\UserRequest;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Mail;
+
+use App\Services\MailService;
 
 class UserController extends Controller
 {
+    public $listuser;
+    protected $mailService;
+    protected $collection;
     /**
      * Display a listing of the resource.
      *
      * @return \Illuminate\Http\Response
      */
+    public function  __construct(MailService $mailService)
+    {
+        $this->mailService = $mailService;
+    }
     public function index()
     {
-        return view('admin.users.index');
+        $this->listuser = session()->get('user');
+        return view('admin.users.index', ['list' => $this->listuser]);
```

Comment:
return view('admin.users.index', ['list' => $this->listuser]);

=>

return view('admin.users.index', ['users' => collection(session()->get('users')));